src/pages/Groups/Group2/Screen1.tsx

```diff
@@ -1,7 +1,66 @@
 import React from 'react'
+import {
+  MainContainer,
+  Screen,
+  BottomNavBar,
+  SearchBar,
+  SearchButton,
+  TextSection,
+  Title,
+  ButtonsSection,
+  PostsSection,
+  FilterBar,
+  Post,
+  ProfilePicture,
+  PostTitle,
+  Author,
+  PostMessage,
+} from './styles/Screen1.styled'
+
+import profilePic from '../../../assets/Group2/cats.jpg'
 
 const Screen1 = () => {
-  return <>this is screen 1</>
+  return (
+    <>
```

Comment:
`<></>` is unnecessary if there is a parent container. in this case, you have `<MainContainer></MainContainer>` so do remove it